src/redux/catergories/catergories.js

```diff
@@ -0,0 +1,20 @@
+const checkcategory = 'bookstore/categories/checkcategory';
+
+const initialcategories = {
+  category: [],
+};
+
+// Create Reducer
+export const checkCategory = () => ({
+  type: checkcategory,
+});
+
```

Comment:
- [ OPTIONAL] 
For more clarity you can rename this file to` categoriesSlice.js`